.github/workflows/documentation_on_tags.yml

```
@@ -0,0 +1,53 @@
+name: Build stable documentation
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: "Set up Python 3.11"
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+      - name: "Install pandoc"
+        run: |
+          sudo apt-get install -y pandoc
+      - name: Install sphinx and dependecies
+        run: pip install sphinx==6.2.0 nbsphinx==0.9.1 sphinx-design==0.4.1 pydata-sphinx-theme==0.13.3
+      - name: Install AISee package
+        run: pip install .
+      - name: Build latest documentation
+        run: |
+          cd docs/
+          make copy-examples
+          make html
+      - name: Switch to gh-pages branch
+        run: git checkout gh-pages
+      - name: Copy documentation to the stable folder of gh-pages
+        run: |
+          rm -rf stable/
+          mkdir stable/
+          cp -r docs/build/html/* stable
+      - name: Copy documentation to the tag folder of gh-pages
+        run: |
+          mkdir ${{ env.RELEASE_VERSION }}/
+          cp -r docs/build/html/* ${{ env.RELEASE_VERSION }}
+      - name: Commit and push dev documentation
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          git add stable/*
+          git add ${{ env.RELEASE_VERSION }}/*
+          git commit -m "Update documentation"
```

```suggestion
          git commit -m "Add ${{ env.RELEASE_VERSION }} documentation"
```

That would be better in the long term (maybe the syntax is not correct)